django_password_validators/password_history/password_validation.py

```diff
@@ -12,11 +12,14 @@
 
 class UniquePasswordsValidator(object):
     """
-    Validate whether the password was once used by the user.
-
+    Validate whether the password was once used by the user
+    in a determined range.
     The password is only checked for an existing user.
     """
 
+    def __init__(self, lookup_range=float('inf')):
+        self.lookup_range = lookup_range if lookup_range >= 0 else float('inf')
```

Comment:
@spokerman12, what if lookup range value provided is not even a number in the first place?

Reply:
Django should handle the exception. `lookup_range` has to be manually specified on `settings.py` and if it's not a number it's the user's fault, no?